projects/2019_12_14_pkt-whitepaper-website.md

```diff
@@ -0,0 +1,170 @@
+﻿# PKT White Paper and Website
+* Project Name: pkt_whitepaper_website (Project)
+* Project Owner: Alex Lightman 
+* Project Manager: Josh Berger: josh@anode.co 
+* Project participants:
+ * Alex Lightman 
+ * Josh Berger
+ * Jesse Berger
+  * Other participants will be added as needed
+* Projected duration: 6 month
+* Projected effort: 8 person-months
+* Requested PKT contribution: PKT50,015,000
+* Project status: **Submitted**
+
+<br/>
+
+## Project summary
+
+After seeing the best and worst of both the Internet and of cryptocurrency projects, I am more excited about the potential of PKT Cash than I have been about any project or protocol in years.
+The objective of this Project is to help the PKT project become more accessible to existing and potential community members by writing a detailed white paper and building a well-designed website where people can gain a deeper understanding of the PKT blockchain and ecosystem. If we are going to reach scale, there needs to be in-depth information on how PKT works and how to use it for people of all technical backgrounds. 
+
+I humbly request the support of the Network Stewards so that I can devote my time, talent, connections, mindshare, and reputation to supporting and enabling the PKT Cash community to achieve its visionary ambitions with easily accessible writing and video for people everywhere to join in. 
+
+The first step towards broad awareness is a white paper. Once the white paper is completed it will require a website to access it. This Project proposes for me as a domain expert to write the PKT white paper, and to coordinate with a web team to develop a dynamic website for interested parties to access and grasp the inner workings of PKT. With a white paper and website in place, the PKT community can begin to scale to achieve its vision. 
+ 
+### Who am I and Why me?
+My name is Alex Lightman and I am a futurist and technologist for almost 40 years. PKT is a unique hybrid of telecommunications protocol, community, and cryptocurrency, and I have unique expertise in thought leadership, invention, marketing, PR, product and service creation, alliance building, community building, business ecosystem development related to IPv6, 4G wireless broadband, Blockchain, and cryptocurrency. 
+
+A few bullet points about my relevant background:
+
+* Author of 16 crypto white papers, the kind that not only bring together focused teams, but also keep teams from overpromising and from running afoul of government regulators. 
```

Comment:
@josh-berger thanks for the submission, can you provide some of these white papers and links to corresponding projects as reference material for this application?

Reply:
Please the white papers Alex Lightman has authored. I have selected his most noteworthy ones in the folder and included another subfolder with his full list of white papers to demonstrate the knowledge and experience he possesses: https://drive.google.com/drive/folders/125W2aHCkjK8sAtXNArOIEKCFdmaIz4iJ?usp=sharing

Reply:
These look fantastic.